addons/website_slides/tests/test_ui_wslides.py

```diff
@@ -144,6 +145,32 @@ def test_course_reviews_elearning_officer(self):
 
         self.start_tour('/slides', 'course_reviews', login=user_demo.login)
 
+    @mute_logger("odoo.http", "odoo.addons.base.models.ir_rule", "werkzeug")
+    def test_course_access(self):
```

Comment:
Maybe we could be a bit more precise in naming ?
```suggestion
    def test_course_access_fail_redirection(self):
```

and add a small docstring to explain what we are testing :)